src/sparql/owldef-self-reference-violation.sparql

```diff
@@ -0,0 +1,12 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX oio: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?term WHERE {
+  { ?term owl:equivalentClass [ owl:intersectionOf [ rdf:rest*/rdf:first ?term ] ] }
+    UNION
+  { ?term owl:equivalentClass [ owl:intersectionOf [ rdf:rest*/rdf:first [ owl:someValuesFrom ?term ] ] ] }
+  FILTER(isIRI(?term) && (STRSTARTS(str(?term), "https://www.inf.ufrgs.br/bdi/ontologies/GEOCORE_")))
```

Comment:
is `https://www.inf.ufrgs.br/bdi/ontologies/GEOCORE_` the correct uri prefix for your classes?

Reply:
I'm not sure I quite understand the doubt. We changed the uri's to be "https://www.inf.ufrgs.br/bdi/ontologies/GEOCORE_xxxxxxx" (7 digits). Does that answer your question?

Reply:
No doubt, just a question. Sounds good then!